src/tests/system/tests/test_cache.py

```diff
@@ -412,3 +414,83 @@ def test_cache__both_ldap_user_email_and_extra_attribute_email_are_stored(client
     assert user_dict["description"] == ["gecos1"], "attribute 'description' was not correct"
     assert user_dict["mail"] == ["user1@example.test"], "attribute 'mail' was not correct"
     assert user_dict["email"] == ["user1@example.test"], "attribute 'email' was not correct"
+
+
+@pytest.mark.parametrize(
+    "type, name",
+    [("group", "grp_nowait"), ("netgroup", "netgrp_nowait")],
+)
+@pytest.mark.importance("low")
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+def test_netgroup__sssd_entry_cache_nowait_percentage(client: Client, provider: GenericProvider, type: str, name: str):
+    """
+    :title: Test SSSD group caching with 'entry_cache_nowait_percentage'
+    :setup:
+        1. Create a test user
+        2. Configure SSSD with:
+            - NSS: filter root groups/users, debug level 9, entry cache nowait percentage set to 50%
+            - Domain 'test': entry cache timeout set to 30 seconds
+        3. Restart SSSD with a clean state
+    :steps:
+        1. Create a group or netgroup named based on the 'type' parameter
```

Comment:
This should be in setup, group creation is not something that needs to be asserted as part of the test. It's required to run the test, but not something the test checks.